qualtran/cirq_interop/_interop_qubit_manager_test.py

```diff
@@ -0,0 +1,30 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import cirq
+import pytest
+
+from qualtran.cirq_interop._interop_qubit_manager import InteropQubitManager
+
+
+def test_interop_qubit_manager():
```

Comment:
Is it worth adding a gate which uses this (like the and / and^ example from the issue?)

Reply:
Adding as a unit test I mean

Reply:
added a test. Merging now.